src/test/resources/sql/with/7a622168.sql

```diff
@@ -0,0 +1,3 @@
+-- file:create_table_like.sql ln:37 expect:true
```

Comment:
See comment below on making sure we're consistent about how we generate these tests.